src/main/kotlin/org/rust/cargo/util/CargoArgsParser.kt

```diff
@@ -47,11 +49,15 @@ class CargoArgsParser private constructor(
             .takeWhile { !OPTION_NAME_RE.matches(it) }
             .take(maxArgsCount)
             .count()
-        check(optionArgsCount >= minArgsCount) { "Too few arguments for option `$option`" }
+        if (optionArgsCount < minArgsCount) {
+            LOG.warn("Too few arguments for option `$option`")
```

Comment:
`getActualOptionArgsCount` is called from `getBuildConfiguration`, which is called from `canRun` methods of our runners (e.g. `org.rust.cargo.runconfig.RsExecutableRunner#canRun`). So this warning will be printed so many times.

Also, it's not a very good idea to do such computations during `canRun` which is called every moment.